shard.go

```diff
@@ -180,14 +179,31 @@ func marshalValues(values map[uint8]interface{}) []byte {
 		}
 
 		// Encode value after field id.
-		// TODO: Support non-float types.
 		switch v := v.(type) {
 		case float64:
+			buf = make([]byte, 9)
 			binary.BigEndian.PutUint64(buf[1:9], math.Float64bits(v))
+		case bool:
+			buf = make([]byte, 2)
```

Comment:
Ah -- OK. I do recall @benbjohnson remarking that `ReadSeries` was debug. If casting strictly happens at the higher level, yeah, then at this low-level it really is just a bunch of bytes.

Let me take another pass.